doc/source/ray-overview/pip_freeze_ray-py39-cpu.txt

```diff
@@ -121,7 +121,7 @@ python-dotenv==1.0.0
 pytz==2022.7.1
 PyWavelets==1.4.1
 PyYAML==6.0.1
-ray @ file:///home/ray/ray-3.0.0.dev0-cp39-cp39-manylinux2014_x86_64.whl#sha256=85cf93d21b91ecef4cf6935194d03ca0c3f907d76267ecd5ad1c6fa6c768717b
+ray @ file:///home/ray/ray-2.9.0-cp39-cp39-manylinux2014_x86_64.whl#sha256=85cf93d21b91ecef4cf6935194d03ca0c3f907d76267ecd5ad1c6fa6c768717b
```

Comment:
Do we need to revert this change, or wait until some later step in the process and then update the `sha` value?

Reply:
yes, please revert.